Src/Plugins/SensitiveTerms/SEC106.SensitiveTerms.json

```diff
@@ -0,0 +1,138 @@
+{
+  "ValidatorsAssemblyName": "SensitiveTerms.dll",
+  "SharedStringsFileName": "SensitiveTerms.SharedStrings.txt",
+  "Definitions": [
+    {
+      "Id": "SEC106/001",
```

Comment:
Do not set the ID here, it is unnecessary: each match expression already sets its id.